test/lint/lint-circular-dependencies.sh

```diff
@@ -32,6 +35,7 @@ EXPECTED_CIRCULAR_DEPENDENCIES=(
     "injector -> staking/transactionpicker -> miner -> injector"
     "injector -> txpool -> txmempool -> injector"
     "keystore -> script/ismine -> keystore"
+    "keystore -> script/sign -> keystore"
```

Comment:
Obviously this pull request merely surfaces this as it activates the lint and that is the current state the repo/code.